lib/gui/pages/main/styles/_main.scss

```diff
@@ -18,7 +18,16 @@
   margin-top: 75px;
 }
 
-.svg-icon[disabled] path {
+svg {
+  width: 100%;
+  height: 100%;
+}
+
+div.center-block > svg-icon {
+  display: inline-block;
+}
+
+div[disabled] > svg-icon path {
```

Comment:
We could implement the disabled property on the React component, but we'd have to do that for every component. To get around it we could make a 'core' component which implements all the 'convenience' functionality we'd expect, and extend any other new classes with that.